README.md

```diff
@@ -133,12 +133,16 @@ Allows the creation of an initial local state for an entity. Note this is essent
 
 ### Entity updates
 
-Allows different kind of updates to be prepared and applied to an entity. For example, a `demandB` must be submitted via a `creation` action.
+Allows different kind of updates to be prepared and applied to an entity. For example, a `demandB` or `match2` must be submitted via a `creation` action.
 
 - `POST /v1/demandB/{demandBId}/creation` - create a creation `creation` transaction and send it to the blockchain.
 - `GET /v1/demandB/{demandBId}/creation` - list a demandB's `creation` transactions and their status.
 - `GET /v1/demandB/{demandBId}/creation/{creationId}` - get the details of a demandB `creation` transaction.
 
+- `POST/v1/match2/{match2Id}/cancellation` - submits cancellation request and creates a transaction that is ready to be synced up 
+- `GET /v1/match2/{match2Id}/cancellation` - retrieves all cancellation transactions for `match2Id`
+- `GET /v1/match2/{match2Id}/cancellation/{cancellationId}` - retrieves a specific cancellation transaction containing more details about it's state
```

Comment:
more details about its state ...not it's ...I think

Reply:
wanted to fill in the blanks

Reply:
haha reasoning is fine - I was talking about grammar. I liked the way it was written - but I think it should have been '...retrieves a specific cancellation transaction containing more details about ITS state'